### PR TITLE
enhancement: include model registry namespace as part of the uninstalling

### PR DESCRIPTION
**Pull request by CFSNM**
there is a new namespace created downstream for the model registry named rhoai-model-registries. Deleting it as part of the uninstall process

### CODE REVIEW

**Review by aloganat — APPROVED** [review]

File: ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot [hunk 1]
```diff
@@ -110,7 +110,10 @@ Uninstall RHODS V2
     ${return_code}    ${output}    Run And Return Rc And Output
     ...    oc delete operatorgroup --all -n ${OPERATOR_NAMESPACE} --ignore-not-found
     Should Be Equal As Integers  ${return_code}   0   msg=Error deleting operatorgroup
-    Log To Console    message=Deleting Operator and it's associate namepsace
+    Log To Console    message=Deleting model registry associated namespace ${MODEL_REGISTRY_NAMESPACE}
+    ${return_code}    ${output}    Run And Return Rc And Output    oc delete ns ${MODEL_REGISTRY_NAMESPACE} --ignore-not-found
```
Comment (jstourac) [inline ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:114]: Why is this needed here and the other namespaces don't have to be deleted separately here?

Comment (CFSNM) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:114]: the model registry namespace does not include the label opendatahub.io/generated-namespace so its not deleted by the command in line 117

Comment (jstourac) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:114]: And is it by design? :thinking:

Comment (CFSNM) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:114]: let me ask the platform devs

Comment (CFSNM) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:114]: @jstourac I am going to close this PR and revert the other one. This namespace is not intended to be deleted when uninstalling the operator, because there can be some model registry instances that need to be kept even if we delete the operator
